libr/fs/shell.c

```diff
@@ -265,12 +265,29 @@ R_API int r_fs_shell_prompt(RFSShell* shell, RFS* fs, const char* root) {
 			} else {
 				char *f = r_str_newf ("./%s", input);
 				if (!r_fs_dir_dump (fs, s, f)) {
-					eprintf ("Cannot open file\n");
+					R_LOG_ERROR ("Cannot open file");
 				}
 				free (f);
 			}
 			free (s);
-		} else if (!memcmp (buf, "help", 4) || !strcmp (buf, "?")) {
+		} else if (r_str_startswith (buf, "o ") || r_str_startswith (buf, "open ")) {
+			input = r_str_nextword (buf, ' ');
+			input = (char *)r_str_trim_head_ro (input);
+			file = r_fs_open (fs, input, false);
+			if (file) {
+				r_fs_read (fs, file, 0, file->size);
+				char *uri = r_str_newf ("malloc://%d", file->size);
+				RIODesc *fd = r_io_open (fs->iob.io, uri, R_PERM_RW, 0);
```

Comment:
rfs doesnt link to r_io. therefor you cant use io apis from the fs module

```
shell.c.obj : error LNK2019: unresolved external symbol r_io_open referenced in function r_fs_shell_prompt
[1107](https://github.com/radareorg/radare2/runs/6964132562?check_suite_focus=true#step:8:1108)
shell.c.obj : error LNK2019: unresolved external symbol r_io_desc_write referenced in function r_fs_shell_prompt
```

you can use corebind or iobind callbacks instead of calling the apis